test/unit/auth/token-verifier.spec.ts

```diff
@@ -535,6 +540,45 @@ describe('FirebaseTokenVerifier', () => {
         });
     });
 
+    it('should decode an unsigned token when using the emulator', async () => {
```

Comment:
Same here. Need some tests via the Auth interface.

Also need to test for other rpc calling APIs like `getUser()` and verify that they behave as expected in the emulator mode.